ci-operator/infra/openshift/origin/artifacts-release-4.1.yaml

```diff
@@ -46,7 +46,7 @@ items:
       spec:
         containers:
         - name: srv
-          image: registry.svc.ci.openshift.org/openshift/origin-v4.1:artifacts
+          image: registry.svc.ci.openshift.org/origin/4.1:artifacts
```

Comment:
Use ocp, otherwise your tests will be artificially delayed from CI

Reply:
Fixed